assets/js/interaction/interactive_submission.js

```diff
@@ -0,0 +1,12 @@
+import domready from 'mf-js/modules/dom/ready';
+
+
+function callback() {
+    let form = document.getElementById( 'search_engine_form' )
+    form.submit();
+}
+
+domready( () => {
+    let form = document.getElementById( 'search_engine_form' )
+    form.addEventListener('change', callback, false);
```

Comment:
```suggestion
    form.addEventListener('change', () => { this.submit() }, false);
```
Je me trompe peut-être ici, mais je pense qu'il y a plus simple que de rechercher le même élément deux fois d'affilée.

Reply:
Non non tout a fait! Je faisais ça car je rentrais dans une boucle infinie. 
Mais c'est mieux comme ça !